src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -387,73 +438,210 @@ public void testTestPermissions() {
       fail();
-    } catch (ResourceManagerException e) {
-      assertTrue(e.getMessage().contains("Project partial-project not found"));
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
     }
   }
 
   @Test
   public void testRuntimeException() {
-    ResourceManagerRpcFactory rpcFactoryMock = EasyMock.createMock(ResourceManagerRpcFactory.class);
-    ResourceManagerRpc resourceManagerRpcMock = EasyMock.createMock(ResourceManagerRpc.class);
-    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(ResourceManagerOptions.class)))
-        .andReturn(resourceManagerRpcMock);
-    EasyMock.replay(rpcFactoryMock);
-    ResourceManager resourceManagerMock =
-        ResourceManagerOptions.newBuilder()
-            .setServiceRpcFactory(rpcFactoryMock)
-            .build()
-            .getService();
     String exceptionMessage = "Artificial runtime exception";
-    EasyMock.expect(resourceManagerRpcMock.get(PARTIAL_PROJECT.getProjectId(), EMPTY_RPC_OPTIONS))
-        .andThrow(new RuntimeException(exceptionMessage));
-    EasyMock.replay(resourceManagerRpcMock);
+    Mockito.doThrow(new RuntimeException(exceptionMessage))
+        .when(resourceManagerMock)
+        .get(PARTIAL_PROJECT.getProjectId());
     try {
       resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
       fail();
-    } catch (ResourceManagerException exception) {
-      assertEquals(exceptionMessage, exception.getCause().getMessage());
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testClearOrgPolicy() {
+    String exceptionMessage = "Should fail because the organization policy doesn't exist.";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .clearOrgPolicy(RESOURCE, ORG_POLICY_INFO);
+    try {
+      resourceManagerMock.clearOrgPolicy(RESOURCE, ORG_POLICY_INFO);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testGetEffectiveOrgPolicy() {
+    Mockito.when(resourceManagerMock.getEffectiveOrgPolicy(RESOURCE, CONSTRAINTS))
+        .thenReturn(ORG_POLICY_INFO);
+    OrgPolicyInfo policyInfo = resourceManagerMock.getEffectiveOrgPolicy(RESOURCE, CONSTRAINTS);
+    assertEquals(CONSTRAINTS, policyInfo.getConstraint());
+    assertEquals(BOOLEAN_POLICY, policyInfo.getBoolPolicy());
+    assertEquals(ETAG, policyInfo.getEtag());
+    assertEquals(LIST_POLICY, policyInfo.getPolicies());
+    assertEquals(UPDATE_TIME, policyInfo.getUpdateTime());
+    assertEquals(VERSION, policyInfo.getVersion());
+  }
+
+  @Test
+  public void testGetEffectiveOrgPolicyWithException() {
+    String exceptionMessage = "Not Found";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
```

Comment:
You've mocked out the method you're trying to test.

Reply:
done